src/apm.js

```diff
@@ -17,21 +17,78 @@
  * under the License.
  */
 
-const { existsSync } = require('fs');
 const { join } = require('path');
-const { name, version } = require('../package.json');
+const { readFileSync } = require('fs');
+const { execSync } = require('child_process');
+const merge = require('lodash.merge');
+const { name, version, build } = require('../package.json');
 
-module.exports = function(serviceName = name) {
-  if (process.env.kbnWorkerType === 'optmzr') return;
+const ROOT_DIR = join(__dirname, '..');
+
+function gitRev() {
+  try {
+    return execSync('git rev-parse --short HEAD', { encoding: 'utf-8' }).trim();
```

Comment:
The way this is done has been updated in `master`. See #52878 to make sure you apply the same change here.

Reply:
Good catch, fixed it now.